peekingduck/configs/model/efficientdet.yml

```diff
@@ -36,3 +36,4 @@ model_format: tensorflow
 model_type: 0 # 0-4
 detect: [0]
 score_threshold: 0.3
+use_jit: true
```

Comment:
Would like to suggest setting the default to `False` so that it does not force install `numba` currently. Then:
1. There should be a documentation to indicate this setting as a user-configurable optimisation.
2. `numba` to be installed only if user sets this to `True`.

Reply:
Set default to false and added documentation for `use_jit` in the node's docstring